description: 'Drivers: hv: vmbus: Use after free in __vmbus_open()'
references:
- https://lore.kernel.org/linux-cve-announce/2024022840-CVE-2021-47049-5cc6@gregkh/T/#u
- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-47049
- https://git.kernel.org/linus/3e9bf43f7f7a46f21ec071cb47be92d0874c48da
- https://git.kernel.org/stable/c/d5c7b42c9f56ca46b286daa537d181bd7f69214f
- https://git.kernel.org/stable/c/f37dd5d1b5d38a79a4f7b8dd7bbb705505f05560
- https://git.kernel.org/stable/c/2728f289b3270b0e273292b46c534421a33bbfd5
- https://git.kernel.org/stable/c/3e9bf43f7f7a46f21ec071cb47be92d0874c48da
- https://www.cve.org/CVERecord?id=CVE-2021-47049
comments:
  debian/carnil: |-
    Introduced in 6f3d791f3006 ("Drivers: hv: vmbus: Fix rescind handling issues").
    Vulnerable versions: 4.13.9 4.14-rc1.
  masami: "Introduced by commit 6f3d791 (\"Drivers: hv: vmbus: Fix rescind handling\
    \ issues\") \nin 4.14-rc1. This commit is not backported to older stable kernels.\
    \ Fixed in 5.13-rc1."
introduced-by:
  mainline: [6f3d791f300618caf82a2be0c27456edd76d5164]
fixed-by:
  cip/5.10: [d5c7b42c9f56ca46b286daa537d181bd7f69214f]
  cip/5.10-rt: [d5c7b42c9f56ca46b286daa537d181bd7f69214f]
  mainline: [3e9bf43f7f7a46f21ec071cb47be92d0874c48da]
  stable/5.10: [d5c7b42c9f56ca46b286daa537d181bd7f69214f]
  ubuntu/groovy: [abe55cfe467d0aa1f6aa5d9bff2b4c699a40075d]
